### PR TITLE
C54009: Asterisk issue

### PR DESCRIPTION
**Pull request by v-makoud**
Hello, @rpetrusha,
Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version. 
Please review and merge the proposed file change to fix to target versions. If you make related fix in another PR  then share your PR number so we can confirm and close this PR.
Many thanks in advance.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/standard/asynchronous-programming-patterns/blocking-application-execution-by-ending-an-async-operation.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Applications that cannot continue to do other work while waiting for the results
   
 -   Use the <xref:System.IAsyncResult.AsyncWaitHandle%2A> property of the <xref:System.IAsyncResult> returned by the asynchronous operation's **Begin***OperationName* method. For an example that demonstrates this approach, see [Blocking Application Execution Using an AsyncWaitHandle](../../../docs/standard/asynchronous-programming-patterns/blocking-application-execution-using-an-asyncwaithandle.md).
```
Comment (mairaw) [inline docs/standard/asynchronous-programming-patterns/blocking-application-execution-by-ending-an-async-operation.md]: I imagine you'd have the same issue here, no?

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good to me @v-makoud but I'm wondering if you also need to make the same change on the previous paragraph.

**Comment by v-makoud** [comment]
thanks @mairaw, I think so, but let me just check with the bug fixing team to be sure

**Comment by v-makoud** [comment]
they agree, i made the change

**Review by mairaw — APPROVED** [review]

**Pull request merged**